ta/tpm_log_test/ta_entry.c

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2020, ARM Limited. All rights reserved.
+ */
+
+#include <ta_tpm_log.h>
+#include <string.h>
```

Comment:
more few lines below.

Reply:
todo:
```c
#include <inttypes.h>
#include <pta_system.h>
#include <string.h>
#include <ta_tpm_log.h>
#include <tee_internal_api.h>

Reply:
Sorry Etienne, I don't quite understand what do you want me to do there. Just rearrange the includes on that order?

Cheers,
Javier

Reply:
sorry, we prefer alphabetical sorting in includes, hence to proposed ordering.

Reply:
Perfect, I get it now.

Thanks,
Javier